lib/win32/taskscheduler/time_calc_helper.rb

```diff
@@ -112,20 +111,21 @@ def time_details(time_str)
           # Replacing strings
           if dt.to_s != ""
             dt["Y"] = "year" if dt["Y"]; dt["M"] = "month" if dt["M"]; dt["D"] = "day" if dt["D"]
-            dt_tm_array_to_hash(dt, tm_detail)
+            dt_tm_string_to_hash(dt, tm_detail)
           end
 
           if tm.to_s != ""
             tm["H"] = "hour" if tm["H"]; tm["M"] = "min" if tm["M"]; tm["S"] = "sec" if tm["S"]
-            dt_tm_array_to_hash(tm, tm_detail)
+            dt_tm_string_to_hash(tm, tm_detail)
           end
         end
         tm_detail
       end
 
       # Method to convert date/time array to hash
```

Comment:
@Nimesh-Msys I think the comment should be here that convert date/time string to hash.

Reply:
Yes, Corrected. I have also updated the entire file as per YARD-DOC format.
Thanks